src/components/DownloadDataButton.js

```diff
@@ -0,0 +1,86 @@
+// Download button for individual plots
+import { saveAs } from "file-saver";
+import React from "react";
+import { Button } from "react-bootstrap";
+import "./DownloadButton.css";
+
+import JSZip from "jszip";
+import JSZipUtils from "jszip-utils";
+
+export default function DownloadDataButton({
```

Comment:
A lot of the code here is duplicated in `DownloadPlotsButton.js` so it'd be good to extract it into a separate file with a function like: 
```
DownloadButton(folder_name, extension)
```

This can be done at a separate PR.